tests/test_transforms.py

```diff
@@ -0,0 +1,67 @@
+"""Tests for transforms.py module."""
```

Comment:
Would you add tests for expected results of some instances ?

Reply:
Good idea, I can add something